tests/__init__.py

```diff
@@ -1 +0,0 @@
-"""Required to ignore mypy errors in the tests folder."""
```

Comment:
why are they not needed anymore

Reply:
True, this is actually something that's debatable. They now just serve as a comment for why these files are there, but I thought it bothersome to have that line in each file `__init__.py` file under `tests`.

What do you think? Keep that line in or remove them from all those `__init__.py` files? And if not such a comment, explain in the documentation?

It's a pity this is just caused by the needs of `mypy`...

Reply:
Why we need those `__init__.py` in the first place. This flag won't help https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-namespace-packages?

Reply:
Good idea! I just tried setting `namespace_packages = True` in the `tox.ini:mypy` section. However, `namespace_packages` makes `mypy` ignores the option
```ini
[mypy-tests.*]
disallow_untyped_defs = False
```
That's why the text:
```python
"""Required to ignore mypy errors in the tests folder."""
```
made sense, actually. So I'll put those docstrings back.